### ISSUE TITLE
"Recover bricked titan" section not accurate -- Unihertz ROM issue?

### ISSUE THREAD

**Issue by neonspectra** [SolidHal/Unihertz-Titan-lineageos-microg#2]
Not sure if I'm just being an idiot here or if Unihertz messed something up with their image files on their Google drive.

I grabbed `2020080711_g61v71c2k_dfl_tee.zip` from [the Unihertz Google Drive](https://drive.google.com/drive/folders/1fpO65z2_r9zT8UImuV8-5JCRBy33yZcV) and extracted the .zip file to its own folder.

However, the `MT6771_Android_scatter.txt` file that you mentioned isn't present in the .zip provided by Unihertz:
![Screenshot from 2021-05-29 19-53-28](https://user-images.githubusercontent.com/10284121/120088781-c2865b80-c0b9-11eb-93a8-ad99fdd8f03c.png)

The only text file that appears to be a scatter file is a `scatter.txt` file in the zip itself. Trying to use that as the scatter file in the SPI flash tool gives this error:
![Screenshot from 2021-05-29 19-52-25](https://user-images.githubusercontent.com/10284121/120088792-dfbb2a00-c0b9-11eb-9728-22376372c32a.png)

I've tried downloading [Unihertz's version of the flash tool](https://drive.google.com/drive/folders/0By1nhWOmuw2KQ2lyNFIybnJIcEE) as hosted by their Google drive and run into the same issue.

---

Like I said, I'm not sure if there's a step missing here or if there's an issue with the files that Unihertz is hosting now. Any thoughts?

**Comment by Dax89** [comment]
I think you are using an old version of SP FlashTool, but I had no luck by flashing the stock firmware even with a recent version.

![Screenshot_20210903_233647](https://user-images.githubusercontent.com/1503603/132067573-b6b35a9a-c9ec-4796-b437-e830ee735c87.png)

In fact I have my Titan bricked and I can't restore it to its stock firmware, I'm pretty sure that you're right, there is an issue with the hosted firmware (the scatter seems wrong).

I've tried to contact them in order to check that firmware with no luck.

**Comment by rumplestilzken** [comment]
> I think you are using an old version of SP FlashTool, and you need the Full Package, not the OTA one: https://drive.google.com/drive/folders/1fpO65z2_r9zT8UImuV8-5JCRBy33yZcV
> 
> But, I had no luck by flashing the stock firmware even with a recent version.
> 
> ![Screenshot_20210903_233647](https://user-images.githubusercontent.com/1503603/132067573-b6b35a9a-c9ec-4796-b437-e830ee735c87.png)
> 
> In fact I have my Titan bricked and I can't restore it to its stock firmware, I'm pretty sure that there is an issue with the hosted firmware (the scatter seems wrong).
> 
> ~I've tried to contact them in order to check that firmware with no luck.~ Firmware for UFS Titans: https://drive.google.com/file/d/1FncSM6FBzJKhrwknEOh3utqkAcMvSfxU/view?usp=sharing

This is correct. If you get an issue you follow the information [here](https://unihertz-titan.neocities.org)
